app/javascript/src/Policies/styles/policies.scss

```diff
@@ -1,5 +1,5 @@
 $icon-font-path: "~bootstrap-sass/assets/fonts/bootstrap/";
-.PolicyConfiguration, .CustomPolicy-form {
+.PolicyConfiguration, .SchemaForm {
   @import "~bootstrap-sass/assets/stylesheets/_bootstrap.scss";
```

Comment:
Why do we need to import all bootstrap?

Reply:
Sadly, the react-jsonschema-form lib depends on the full bootstrap to render the form. They are planning to be agnostic in the future...but not near future.